src/core_plugins/kibana/public/management/sections/indices/edit_index_pattern/indexed_fields_table/indexed_fields_table.js

```diff
@@ -1,22 +1,22 @@
 import _ from 'lodash';
 import 'ui/paginated_table';
-import nameHtml from 'plugins/kibana/management/sections/indices/_field_name.html';
-import typeHtml from 'plugins/kibana/management/sections/indices/_field_type.html';
-import controlsHtml from 'plugins/kibana/management/sections/indices/_field_controls.html';
+import fieldNameHtml from './field_name.html';
```

Comment:
I find it a little easier to read the code when the file names and variable names have consistency, and I felt like this helped add more consistency. Now I can just scan the code for "field name".